shared/elements/gaia_lowstorage_dialog/examples/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
yep; I think dashed-components are for the new ones that come from external repositories, underscore_components are for the ones that are in gaia/shared only, and should be replaced in the future by external ones... So I choose underscore here because it uses an underscore component to keep the old style.